### DISCUSSION TITLE
As a linux user, should I use the flatpak or the distribution package (PPA)?

### DISCUSSION THREAD

**Discussion by BarbzYHOOL** [Floorp-Projects/Floorp#503] [Q&A] [answered]
I installed the flatpak for quick testing, but I feel like switching to the Ubuntu PPA. I suppose it might be better but I'm not even sure

**Comment by CutterKnife** [comment, marked as answer]
The PPA version is maintained by the developers and the Flatpak version by the contributors.

**Reply by BarbzYHOOL** [reply]
oh! so let's go for the PPA

**Reply by BarbzYHOOL** [reply]
Can I simply copy the whole profile directory from the flatpak directory to the home Floorp directory though?

**Comment by grompes** [comment]
RUssia
Делаю по инструкции https://ppa.floorp.app
В итоге ошибка что "E: Невозможно найти пакет floorp" (not found)
Конечно мне не трудно найти и скачать deb, но если уж вы заявили возможность, то будеть добры её отработать. Или тогда удалите ссылки с главной страницы Hub о такой "простой" установке через консоль
Linux mint 21,2 x64

upd:
оказалась разница в строчке https://ppa.floorp.app/Floorp.list --> https://ppa.ablaze.one/Floorp.list

**Reply by Ryosuke-Asano** [reply]
@creeper-0910

**Reply by creeper-0910** [reply]
Please run sudo apt update and try again.

**Reply by grompes** [reply]
Проблема не в sudo (к слову копировал-вставлял как у вас на сайте указано. Если там нет sudo, то добавьте её на сайте), проблема в строке которую я указал в upd "https://ppa.floorp.app/Floorp.list --> https://ppa.ablaze.one/Floorp.list"
В итоге я установил браузер но через другую инструкцию. У вас ребята инструкции на официальныхз сайтах разные. Наведите у себя порядок

**Reply by creeper-0910** [reply]
Hmmm... The files those two url's refer to should be identical...
